core/deployment/src/main/java/io/quarkus/deployment/Feature.java

```diff
@@ -141,6 +141,7 @@
     SPRING_BOOT_PROPERTIES,
     SPRING_CACHE,
     SPRING_CLOUD_CONFIG_CLIENT,
+    SPRING_CLOUD_CONFIG_CLIENT_OIDC,
```

Comment:
We probably don't need this now either, right?

Reply:
Right, sorry for that..